pkg/cmd/testdata/scripts/scriptenv.txtar

```diff
@@ -1,15 +1,22 @@
-[!exec:python3] skip 'python3 not found in $PATH'
+[windows] skip 'UNIX only'
```

Comment:
This *should* work on Windows, per https://learn.microsoft.com/en-us/windows/win32/procthread/inheritance#inheriting-environment-variables.